README.md

```diff
@@ -5,7 +5,7 @@ This repo consists of material for the HiggsML Uncertainty Challenge. The compet
 ***
 
 ## Challenge Website
-The challenge website will appear on Codabench: https://www.codabench.org/. Details about how to join the challenge and the challenge description will be made available on the website
+The challenge website is on [Codabench](https://www.codabench.org/competitions/2977/). Details about how to join the challenge and the challenge description will be made available on the website
```

Comment:
Change `will me made available on the website` to `are available on the website`

Reply:
Thanks @ihsaan-ullah , I fixed it